docs/guides/03-Styling-your-visualization-using-Expressions.md

```diff
@@ -0,0 +1,198 @@
+
+
+<div class="example-map">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/styling/color-spaces.html"
+        width="100%"
+        height="1000"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-color-spaces">View my source code!</a>
+</div>
+
+
+You can take a look at the reference for a detailed description of all color expressions.
+
+#### Transparency, alpha channel, opacity, and filtering
+
+There are multiple ways to set the alpha channel:
+- The alpha channel when using a color constant or color constructor. You should use this whenever you can (i.e., you want to change the alpha channel of a color constant or color constructor). For example: `rgba(255,255,255,0.5)` or `#F375`.
+- Using `opacity` to override the alpha channel. You should use this when the input color expression is not a color constant or color constructor. For example, it is very common to combine `opacity` with `ramp`. For example: `opacity(ramp($price, prism), 0.5)
+- Using the `filter` property. The filter property multiplies the color alpha channel (both fill and stroke) of the features with its value. This, however, has a special semantic regarding viewport aggregation expressions. We'll look at this later.
+
+
+### Numeric and boolean expressions basics
+
+In CARTO VL numeric expressions are a first class citizen. You have access to math like `+`, `-`, `*`,`/`,`^`,`log`,`sin`... out of the box.
+
+You’ll also be able to get boolean-like values within CARTO VL. Some ways to get these “boolean-like” types are comparison functions (`==`, `!=`, `>`, `<`, `>=`), utils (`between`, `in`) and logic functions (`not`, `or`, `and`).
+
+We talk about boolean-like types and not just boolean types because there is no “boolean” type in CARTO VL. Every “boolean-like” type is actually emulated by the numeric type.
+This makes our boolean logic a [fuzzy boolean logic](https://en.wikipedia.org/wiki/Fuzzy_logic), making most animations (blendings) work out of the box.
+
+### Using dataset feature properties
+To refer to a dataset feature property we can use the dollar sign. For example, `$price` refers to the features property named ‘price’ in the dataset.
+
+We’ll use this extensively in the Data-driven visualization guide.
+
+### Accessing and modifying visualization properties
+
+Once a visualization has been created, you can access and modify all the styling properties through the returned [Visualization]().
+
+Once you have a `Visualization` object, you can:
+- Get its property values by using the `.value` getter: `const currentColor = viz.color.value;`
+- Modify its property values by using the`.blendTo()` method: `viz.color.blendTo(‘green’);`
+
+<div class="example-map">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/guides/viz-properties/accessing-viz-properties.html"
```

Comment:
This is an interesting example!. I would just try to change the font for 'The current color is...' to match others and also use .toFixed(0) in the r.g.b components

Reply:
The CSS is not correctly displayed in staging due to a bug in the dev-center.

I'll add ` .toFixed(0) `